Changes for RH Certification (#5)
nginxinc/nginx-ingress-operator: docs/openshift-installation.md

> # Openshift Installation

Reviewer: just to be more specific. This is how this doc is referenced in the docs/README.md
```suggestion
# Installation in an Openshift cluster the OLM.
```

> # nstallation in an Openshift cluster the OLM

Reviewer 2: ```suggestion
# Installation in an Openshift cluster the OLM
```

This installation method is the recommended way for Openshift users. **Note**: Openshift version must be 4.2 or higher.

The NGINX Ingress Operator is a [RedHat certified Operator](https://connect.redhat.com/en/partner-with-us/red-hat-openshift-operator-certification).

> 1. In the Openshift dashboard, click in `Operators` > `Operator Hub` in the left menu and use the search box to type `nginx ingress`:

Reviewer: Note sure if "click in" is correct or common. Click on?

In https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html just "click" is used.

![alt text](./images/openshift1.png "Operators")

> 1. Click in the NGINX Ingress Operator and click Install:

Reviewer: should we highlight the names of menus, buttons, etc in all sections, note only in the previous one?  Perhaps in **bold** or *italic*?

![alt text](./images/openshift2.png "NGINX Ingress Operator")
1. Click in Subscribe (**Note:** We recommend to mark Specific Namespace for the Operator):
![alt text](./images/openshift3.png "NGINX Ingress Operator Install")

Openshift will install the NGINX Ingress Operator:

![alt text](./images/openshift4.png "NGINX Ingress Operator Subscribe")

> You can now deploy the NGINX Ingress Controller instances manually following the [examples](../examples) or using the Openshift Dashboard.

Reviewer: in that example, should we use "oc" tool or make some kind of comment about the "oc" tool. In the OS docs like this one https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html, "oc" is used.

Author: Hmm I think I can add a note?  BTW `kubectl` should work for Openshift as well  (from 4.1 https://docs.openshift.com/container-platform/4.1/cli_reference/usage-oc-kubectl.html#the-kubectl-binary), obviously RH uses `oc` in all their official doc. However the examples were designed for k8s and os, not only os.

Reviewer: @Rulox thanks for the clarification. If OS users are used to and ok with `kubectl` commands, than there is no need for adding notes or changes

> You can now deploy the NGINX Ingress Controller instances manually following the [examples](../examples) or using the Openshift Dashboard.

Reviewer: Is there any link for OpenShift dashboard?

Not sure if "manually" should be used. Does the OpenShift dashboard provide some kind of  non-manual way? 🤔 

```suggestion
You can now deploy the NGINX Ingress Controller instances [using `kubectl`](../examples) or using the Openshift Dashboard.
```

Author: What do you mean by link? To a documentation? or in a cluster? In a cluster it depends on the fqdn of the installation.

Yes, the way of adding CRs in Openshift is using the Openshift dashboard (instead of oc/kubectl). Everything through the dashboard. Makes sense?

Reviewer: yeah, a link to some documentation (ours or external). 

because it wasn't clear what you meant by "using the Openshift Dashboard".

or perhaps we should not mention the dashboard at all, because we don't provide any documentation for that?

Author: Ok will remove the Dashboard part. I think most OS customers are used to do this over the dashboard anyways, so we'll keep this open
